AY2122S1-CS2103T-T11-2/tp
# Fix edit command bug returning unfiltered flashcard list

### Description


Previously, running the edit command on a filtered list would successful edit the correct flashcard, BUT lists all existing flashcards after (instead of remaining in the same filtered list)

This bug was inherited from the original AB3 repo.

Closes #165 

### Type of change


### How to test


1. Filter the flashcard list
2. Edit one of the flashcards in the filtered flashcard list
3. The displayed list of flashcards should remain the same as the filtered list before the command was run (i.e. it doesn't show all existing flashcards anymore)

### Checklist



~- [ ] I have updated the User Guide~
~- [ ] I have updated the Developer Guide~
~- [ ] I have updated my individual project portfolio~
